### PR TITLE
modify fluentd conf, port 20001 for tcp, 20002 for udp, 20003 for http

### PR DESCRIPTION
**Pull request by potonix**

### CODE REVIEW

File: test/send-to-fluentd/20000-fake-1-logging-tcp.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+testlib.send_logs.tcp_connect('localhost', 20000)
+
+# ts = testlib.timeutils.str2ts('2020-01-02T10:40:00.000Z')
+ts = int(time.time())
+
+# s1 = '2020-01-02T18:31:00.000Z'
+# t1 = testlib.timeutils.str2ts(s1)
+# s2 = testlib.timeutils.ts2str(t1)
+
+log = testlib.mock_data.mock_logging_data_20001(ts, 1)
```
Comment (zongzw) [inline test/send-to-fluentd/20000-fake-1-logging-tcp.py:13]: mock_logging_data_20001 doesn't match the 20000 port. Shall we change 20000 to 20001 in `testlib.send_logs.tcp_connect('localhost', 20000)` and the filename?

File: test/send-to-fluentd/20001-fake-1-logging-udp.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import testlib
+import time
+
+testlib.send_logs.udp_connect('localhost', 20001)
```
Comment (zongzw) [inline test/send-to-fluentd/20001-fake-1-logging-udp.py:4]: change to 20002 for udp?

File: conf.d/docker-compose.yml [hunk 3]
```diff
@@ -19,6 +19,7 @@ services:
     container_name: FLUENTD
     ports:
       - "20000-20010:20000-20010/tcp"
+      - 20001:20001/udp
```
Comment (zongzw) [inline conf.d/docker-compose.yml:22]: 20001  is included in 20000-20010, how about  deleting  `/tcp` directly?

Comment (potonix) [reply conf.d/docker-compose.yml:22]: Docker port mapping defaults to tcp, so /tcp can be deleted, but 20001 must be specified as udp.

File: test/send-to-fluentd/20000-fake-1-logging-tcp.py (hunk 1 above)
Comment (potonix) [reply test/send-to-fluentd/20000-fake-1-logging-tcp.py:13]: Get it.

File: test/send-to-fluentd/20001-fake-1-logging-udp.py (hunk 2 above)
Comment (potonix) [reply test/send-to-fluentd/20001-fake-1-logging-udp.py:4]: Ok, and I plan to do another http test script for port 20003 later.

File: conf.d/docker-compose.yml (hunk 3 above)
Comment (zongzw) [reply conf.d/docker-compose.yml:22]: > Docker port mapping defaults to tcp, so `/tcp` can be deleted, but 20001 must be specified as `udp`.

Does it mean if we use `- 20000-20010:20000-20010` only, `udp` on 20001 will not work?

Comment (potonix) [reply conf.d/docker-compose.yml:22]: > > Docker port mapping defaults to tcp, so `/tcp` can be deleted, but 20001 must be specified as `udp`.
> 
> Does it mean if we use `- 20000-20010:20000-20010` only, `udp` on 20001 will not work?

Yes.

Comment (zongzw) [reply conf.d/docker-compose.yml:22]: Got it. Thanks.

**Pull request merged**